core-bundle/src/Command/UserCreateCommand.php

```diff
@@ -0,0 +1,329 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Command;
+
+use Contao\BackendUser;
+use Contao\Config;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\UserGroupModel;
+use Contao\Validator;
+use Doctrine\DBAL\Connection;
+use RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Security\Core\Encoder\EncoderFactoryInterface;
+
+/**
+ * Creates a new Contao back end user.
+ *
+ * @internal
+ */
+class UserCreateCommand extends Command
+{
+    /**
+     * @var ContaoFramework
+     */
+    private $framework;
+
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    /**
+     * @var EncoderFactoryInterface
+     */
+    private $encoderFactory;
+    /**
+     * @var array
+     */
+    private $locales;
+
+    public function __construct(ContaoFramework $framework, Connection $connection, EncoderFactoryInterface $encoderFactory, array $locales)
+    {
+        $this->framework = $framework;
+        $this->connection = $connection;
+        $this->encoderFactory = $encoderFactory;
+        $this->locales = $locales;
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('contao:user:create')
+            ->addOption('username', 'u', InputOption::VALUE_REQUIRED, 'The username')
+            ->addOption('name', null, InputOption::VALUE_REQUIRED, 'The name')
+            ->addOption('email', null, InputOption::VALUE_REQUIRED, 'The email address')
+            ->addOption('password', 'p', InputOption::VALUE_REQUIRED, 'The username')
+            ->addOption('language', null, InputOption::VALUE_REQUIRED, 'The user language')
+            ->addOption('admin', null, InputOption::VALUE_NONE, 'Give new user admin permissions?')
+            ->addOption('groups', null, InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY, 'The groups to add the assign the user to')
+            ->addOption('pwChange', null, InputOption::VALUE_NONE, 'Require password change')
```

Comment:
Camel case seems strange for a CLI option. Maybe `--change-password` instead? Description could be more extensive, e.g. _Require user to change the password on the first back end login`.

Also, maybe this should be the default case? And/or there could be an option to generate a password? Though we sure can add that in a later step.

Reply:
Password generation is a science in its own 😄. We would need to create a password generation password at first.

Reply:
I would prefer `--change-password` or `--chpasswd` as well.